tools/oas-to-am/test/features/step_definitions/custom-steps.sed

```diff
@@ -0,0 +1,3 @@
+# A place to put your own step definitions
```

Comment:
Do we need this file? I don't see any other steps or usage of this. I'd rather link to the apickli docs itself for how to add custom steps.

Reply:
good point - currently this is using an `apickli-shell` project which isn't properly published. this is its first usage. i would prefer to leave this file here until there are good docs to link to.

@seymen can we consider publishing apickli-shell to the apickli org?

Reply:
sure we can.